staynomad/Nomad-Front
# Updated filter values to persist through searches

# Relevant issue

Closes #525

# Summary of change


 - Updated listing filter values to persist through multiple searches
 - Addressed react warnings (triple equals and unused import)

# Testing/Verification

 - Select any filter value using the filter modal (ie. max price, country, price: low to high)
 - Hit confirm and search to display filtered listings
 - Click on the filter icon again. The filter modal should contain the filters that are currently applied
